packages/terra-table/src/TableHeaderCell.jsx

```diff
@@ -4,71 +4,129 @@ import IconDown from 'terra-icon/lib/icon/IconCaretDown';
 import IconUp from 'terra-icon/lib/icon/IconCaretUp';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
-import styles from './Table.module.scss';
+import styles from './TableHeaderCell.module.scss';
+import TableUtils from './TableUtils';
 
 const cx = classNames.bind(styles);
 
-const TableHeaderCellMinWidth = {
-  TINY: 'tiny',
-  SMALL: 'small',
-  MEDIUM: 'medium',
-  LARGE: 'large',
-  HUGE: 'huge',
-};
-
-const TableHeaderCellSort = {
-  ASC: 'asc',
-  DESC: 'desc',
+const ariaSortMap = {
+  asc: 'ascending',
+  desc: 'descending',
+  none: 'none',
 };
 
 const propTypes = {
+  /**
+   * Content to be displayed for the column header.
+   */
+  children: PropTypes.node,
   /**
    * Content to be displayed for the column header
    */
-  content: PropTypes.node.isRequired,
+  icon: PropTypes.element,
```

Comment:
Looks like the icon prop definition didn't get updated

Reply:
Updated.